packages/ui/package.json

```diff
@@ -1,16 +1,15 @@
 {
-  "name": "truffle-ai-frontend",
-  "version": "0.1.0",
+  "name": "ui",
+  "license": "MIT",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
+    "serve": "next start",
```

Comment:
Why rename to serve? The default next command is also start

Reply:
because we use serve everywhere